src/Analytics/Analytics.csproj

```diff
@@ -83,7 +83,7 @@
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetFramework)' == 'net6.0-ios'">
-        <PackageReference Include="Xamarin.Firebase.iOS.Analytics" Version="8.10.0.3" />
+        <PackageReference Include="AdamE.Firebase.iOS.Analytics" Version="10.24.0" />
```

Comment:
On Analytics, besides the updated SDK we need to include the Installations SDK as well. I'm still unsure why it is required, but otherwise, I was getting compilation errors and analytics was not starting for me on my demo project and real project.

```xml
<ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">
        <PackageReference Include="AdamE.Firebase.iOS.Installations" Version="10.24.0" />
        <PackageReference Include="AdamE.Firebase.iOS.Analytics" Version="10.24.0" />
    </ItemGroup>
```

Reply:
PR was closed before I could address this, but you're right. I should probably fix this in the bindings projects.